consensus/core/src/context.rs

```diff
@@ -20,7 +20,7 @@ use consensus_config::{NetworkKeyPair, ProtocolKeyPair};
 /// of this authority.
 #[allow(dead_code)]
 #[derive(Clone)]
-pub(crate) struct Context {
+pub struct Context {
```

Comment:
Are we switching all `pub(crate)` to `pub`? It is fine but we just need to be consistent.